app/src/main/res/layout/popaddwindow.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools" android:background="@drawable/rounded_corners"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="16dp"
+    >
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/codeName"
+        android:layout_width="match_parent"
+        android:layout_height="75dp"
+        android:hint="@string/codename"
```

Comment:
codename is made of two words
in camelCase it would be codeName
in snake_case(the one you're using here) it should be code_name

but rename according to standards